animals_and_tips.html

```diff
@@ -126,115 +126,115 @@ <h2>Hover on each image to discover information about them</h2>
 <div class="container">
 	<img src="images/placeholder.png" alt="img" class="image">	
 	<div class="overlay overlayLeft">
-		<div class="text"><h1>Lion</h1><h2>Panthera Leo</h2><p>Stand your ground, retreating very slowly, continuing to face the lion while clapping your hands, shouting and waving your arms around to make yourself look bigger. Most charges are mock charges, so you will usually be fine. And hold your ground! Never run or turn your back.
+		<div class="text"><h1>Lion</h1><h2>Panthera Leo</h2><p>If in contact with a lion, stand your ground, retreating very slowly, continuing to face the lion while clapping your hands, shouting and waving your arms around to make yourself look bigger. Most charges are mock charges, so you will usually be fine. And hold your ground! Never run or turn your back.
 <p></div>
 	</div>
 </div>
 <div class="container">
 	<img src="images/placeholder.png" alt="img" class="image">	
-	<div class="overlay overlayTop">
-		<div class="text"><h1>Impala</h1><h2>Aepyceros melampus</h2><p>The impala is one of the most common and most graceful of all Africa's antelopes and are no threat to humans. A slender, agile creature, it can clear formidable obstacles and run at speeds faster than 60km/h. Most young impala are born around mid-day as this is the safest time to give birth since most of their enemies are resting.<p></div>
+	<div class="overlay overlayCross">
+		<div class="text"><h1>Impala</h1><h2>Aepyceros melampus</h2><p>The impala is one of the most common and most graceful of all Africa's antelopes and is no threat to humans. A slender, agile creature, it can clear formidable obstacles and run at speeds faster than 60km/h. Most young impalas are born around mid-day as this is the safest time to give birth since most of their enemies are resting.<p></div>
 	</div>
 </div>
 <div class="container">
 	<img src="images/placeholder.png" alt="img" class="image">	
-	<div class="overlay overlayRight">
+	<div class="overlay overlayTop">

```

Comment:
Some of the overlays were switched with a different type, because each type of overlay corresponds to one of three colours, green, orange and yellow, thereby rearranging was done to make sure colours were placed in suitable positions.